CorsixTH/Lua/world.lua

```diff
@@ -1205,13 +1205,8 @@ function World:onEndMonth()
     self:checkIfGameWon()
   end
 
-  -- Change population share for the hospitals, TODO according to reputation.
-  -- Since there are no competitors yet the player's hospital can be considered
-  -- to be fairly good no matter what it looks like, so after gbv.AllocDelay
-  -- months, change the share to half of the new people.
-  if self.month >= self.map.level_config.gbv.AllocDelay then
-    self:getLocalPlayerHospital().population = 0.5
-  end
+  local local_hospital = self:getLocalPlayerHospital();
```

Comment:
After reading the description of AllocDelay I agree.

Should be 0.25 until AllocDelay, and this after.